Source/WebCore/platform/graphics/egl/GLContext.cpp

```diff
@@ -52,10 +49,33 @@
 #include <X11/Xutil.h>
 #endif
 
-#include <wtf/Vector.h>
-
 namespace WebCore {
 
+static ThreadSpecific<GLContext*>& currentContext()
+{
+    static ThreadSpecific<GLContext*>* context;
+    static std::once_flag flag;
+    std::call_once(flag, [] {
+        context = new ThreadSpecific<GLContext*>();
+    });
+    return *context;
+}
+
+static bool initializeOpenGLShimsIfNeeded()
+{
+#if USE(OPENGL_ES) || USE(LIBEPOXY) || (USE(ANGLE) && !(PLATFORM(GTK) || PLATFORM(WPE)))
```

Comment:
`USE(LIBEPOXY)` is guaranteed to be true for both GTK and WPE, so this can be simplified.